sound/soc/sof/intel/hda-dsp.c

```diff
@@ -398,6 +398,9 @@ static int hda_dsp_update_d0i3c_register(struct snd_sof_dev *sdev, u8 value)
 	snd_sof_dsp_update8(sdev, HDA_DSP_HDA_BAR, chip->d0i3_offset,
 			    SOF_HDA_VS_D0I3C_I3, value);
 
+	/* Wait for about 30 us to propagate I3 transition */
```

Comment:
what is an I3 transition?

Reply:
I3 bit in D0i3 register. This bit will transition from original value to new value no more than 30 us. We should check CIP (Command-In-Progress) when this transition happened.

Reply:
CIP is set by HW on a 1->0 or 0->1 transition of bit I3. And CIP is cleared when the transition is complete